src/platform/forms-system/src/js/components/FieldTemplate.jsx

```diff
@@ -30,15 +31,19 @@ export default function FieldTemplate(props) {
   ) : null;
   const label = uiSchema['ui:title'] || props.label || '';
   const isDateField = uiSchema['ui:widget'] === 'date';
-  const showFieldLabel =
-    uiSchema['ui:options'] && uiSchema['ui:options'].showFieldLabel;
-  const useLabelElement = showFieldLabel === 'label';
+  const uiOptions = uiSchema['ui:options'] || {};
+
+  const { showFieldLabel } = uiOptions;
+  const useLabelElement =
+    showFieldLabel === 'label' || showFieldLabel === 'no-wrap';
 
-  const description = uiSchema['ui:description'];
+  // The children may also include the description
+  const description = uiOptions?.hideDuplicateDescription
+    ? null
```

Comment:
Are we saying to set `description` to `null` if the `hideDuplicateDescription` doesn't exist? and if it exists we set it to the `ui:description`?

Reply:
It's the opposite. In a tenary, it's set up as `check ? {true check} : {false check}`; so, `null` is only set if `uiOptions` is defined AND the `hideDuplicateDescription` is `truthy`, otherwise it will render the `ui:description`